setup/ci/index.md

```diff
@@ -22,6 +22,7 @@ independently of one-another, enumerated below.
 
 These are the CI systems currently supported by Spinnaker:
```

Comment:
@dorbin What are our thoughts generally on what it means when we say a "CI system is supported by Spinnaker"? Does it mean "can be integrated with in _some_ way"? Or do we consider support for a CI system to mean a) ability to trigger on completion of CI builds and b) ability to call the CI system *from* a Spinnaker pipeline via a stage?

Reply:
A good question. And I'll evade it for the moment by saying we're not enabling CI for Build in the way we are for Jenkins, Travis, and Wercker. There isn't `hal config ci google enable`.

I think this doc should go in /guides/user/pipeline/triggers/, for now at least. *But*, we should probably include a note on this page saying you can use Build with Spinnaker.  My two ¢.

Reply:
> I think this doc should go in /guides/user/pipeline/triggers/

Most CI systems provide triggers too, though - what then **is** our definition of what a "CI system" adds to Spinnaker? Should Jenkins, Travis etc. (if they can function as triggers) also be mentioned in the [trigger guide](https://www.spinnaker.io/guides/user/pipeline/triggers/)?

Reply:
I would say GCB should be classified as a _supported_ CI system since we can not only trigger off of builds, but also consume artifacts produced by it.

If we want to add `hal` commands to give it the appearance of a typical CI system, that's definitely worth doing and thinking over as we simplify our artifact flows to abstract event streams into trigger types (i.e. a user should be thinking of getting a docker image from GCB, not translating a pub/sub message from a certain subscription).